From: Tomasz Karczewski <tomasz.karczewski@consult.red>
Subject: ONEM-32583: Increase RPC communication timeout to 20 secs

Justification: while similar change is present in older Thunder
versions (4.1) - it was removed by Comcast in 4.2. Turns out
there is one scenario where we still need it - for wpe, where
setting URL in LGI implementation means waiting synchronously until the
page is loaded; if it doesn't happen before this timeout, the
request will be discarded & user gets an error (see
WebKitBrowser/WebKitImplementation.cpp, URL method for
reference & URL_LOAD_RESULT_TIMEOUT_MS contstant value)
Hence, raising the timeout back to 20 sec.

---
 Source/com/Administrator.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Source/com/Administrator.h b/Source/com/Administrator.h
index 24ce6097..bf861e8d 100644
--- a/Source/com/Administrator.h
+++ b/Source/com/Administrator.h
@@ -36,7 +36,7 @@ namespace RPC {
 #ifdef __DEBUG__
     enum { CommunicationTimeOut = Core::infinite }; // Time in ms. Forever
 #else
-    enum { CommunicationTimeOut = 3000 }; // Time in ms. 3 Seconds
+    enum { CommunicationTimeOut = 20000 }; // Time in ms. 20 Seconds
 #endif
     enum { CommunicationBufferSize = 8120 }; // 8K :-)

